rules/framework/vfs_overlay.bzl

```diff
@@ -268,6 +289,7 @@ framework_vfs_overlay = rule(
         "extra_search_paths": attr.string(mandatory = False),
         "has_swift": attr.bool(default = False),
         "modulemap": attr.label(allow_single_file = True),
+        "swiftmodule_contents": attr.label_list(allow_files = True),
```

Comment:
add comment: pass anything under .swiftmodule here

Reply:
+1 - one ohter thing is that we'd want a default here. If a dev is already using the rule, then the new attribute would be a breaking change for them. Minor and simple fix - it's done in the other attrs BTW